azkaban-web-server/src/web/js/azkaban/view/flow-execution-list.js

```diff
@@ -413,6 +416,13 @@ azkaban.ExecutionListView = Backbone.View.extend({
     }
 
     if (node.type != 'flow' && node.status != 'SKIPPED') {
+      if (node.cluster != null) {
+        var clusterLink = document.createElement("a");
+        $(clusterLink).attr("href", node.cluster);
+        $(clusterLink).text("cluster");
```

Comment:
Can you please use capital letter c?
What will happen for CANCEL or disabled statuses? Can you please test for all the statuses and see cluster link? Please share screenshots.